src/FakeItEasy/FakeItEasy.nuspec

```diff
@@ -34,7 +34,7 @@
         <dependency id="System.Reflection.Extensions" version="4.0.1" />
         <dependency id="System.Reflection.TypeExtensions" version="4.1.0" />
         <dependency id="System.Runtime.Loader" version="4.0.0" />
-        <dependency id="Castle.Core" version="4.0.0-alpha001" />
+        <dependency id="Castle.Core" version="4.0.0-beta001" />

```

Comment:
Consistent, but I'd rather see this move to 3.3.3. I can work on a commit that does this.